### DISCUSSION TITLE
How to manage white balance in Pinta?

### DISCUSSION THREAD

**Discussion by droumanet** [PintaProject/Pinta#1191] [Q&A]
Hi there,
some pictures are yellowish and I would to find the original color.
I don't know how to do this with Pinta.
Thanks.

**Comment by cameronwhite** [comment]
I think the Adjustments menu will likely have something you can use - e.g. the Curves or Hue/Saturation adjustments could be used for re-coloring an image
https://www.pinta-project.com/user-guide/colors/
